src/encoded/static/components/filegallery.js

```diff
@@ -505,8 +526,8 @@ class RawSequencingTable extends React.Component {
             // Make lists of files that are and aren't paired. Paired files with missing partners
             // count as not paired. Files with more than one biological replicate don't count as
             // paired.
-            const nonpairedFiles = [];
-            const pairedFiles = _(files).filter((file) => {
+            let nonpairedFiles = [];
```

Comment:
Why did you change this from "const" to "let"?

Reply:
nonpairedFiles gets reassigned on line 573. ESLint gives me an error.

Reply:
Thanks. I missed that.